package.json

```diff
@@ -34,7 +34,7 @@
     "prettier:check": "prettier --list-different '@(packages|scripts)/**/*.@(css|graphql|js)' '*.js'",
     "stage:venia": "yarn venia run start",
     "stats:venia": "yarn venia run build:analyze && yarn venia run stats",
-    "storybook:venia": "yarn venia run storybook",
+    "storybook:venia": "pushd ./packages/venia-ui && yarn storybook && popd",
```

Comment:
Haha tricky. How about we just use the workspace command to run the command? `yarn workspace @magento/venia-ui run storybook`. I think you made this change because `yarn venia` correlates to `yarn workspace @magento/venia-concept` instead of `venia-ui`.

Reply:
Great idea haha, I'll update with this.